browser/brave_browser_main_parts.h

```diff
@@ -10,9 +10,7 @@
 
 #include "chrome/browser/chrome_browser_main.h"
 
-#if BUILDFLAG(IS_WIN)
```

Comment:
in other places it is `#if BUILDFLAG(IS_WIN) || BUILDFLAG(IS_ANDROID)`, shouldn't we be consistent here?

Reply:
++